.circleci/config.yml

```diff
@@ -1,86 +1,86 @@
 version: 2
 
 defaults: &defaults
-    working_directory: ~/repo
-    docker:
-        - image: circleci/node:8.9.1
+  working_directory: ~/repo
+  docker:
+    - image: circleci/node:8.9.1
 
 jobs:
-    build:
-        <<: *defaults
-        steps:
-            - checkout
-
-            - restore_cache:
-                  keys:
-                      - v1-dependencies-{{ checksum "package.json" }}
-                      - v1-dependencies-
-
-            - run: npm install
-            - run: sudo npm install typescript -g
-            - run:
-                  name: Build project
-                  command: tsc
-
-            - save_cache:
-                  paths:
-                      - node_modules
-                  key: v1-dependencies-{{ checksum "package.json" }}
-
-            - persist_to_workspace:
-                  root: ~/repo
-                  paths: .
-
-    lint:
-        <<: *defaults
-
-        steps:
-            - attach_workspace:
-                  at: ~/repo
-
-            - restore_cache:
-                  keys:
-                      - v1-dependencies-{{ checksum "package.json" }}
-                      - v1-dependencies-
-
-            - run:
-                  name: Run linter
-                  command: npm run tslint:check
-
-            - save_cache:
-                  paths:
-                      - node_modules
-                  key: v1-dependencies-{{ checksum "package.json" }}
-
-    test:
-        <<: *defaults
-        steps:
-            - attach_workspace:
-                  at: ~/repo
-
-            - restore_cache:
-                  keys:
-                      - v1-dependencies-{{ checksum "package.json" }}
-                      - v1-dependencies-
-
-            - run:
-                  name: Run tests
-                  command: npm run test:ci
-
-            - save_cache:
-                  paths:
-                      - node_modules
-                  key: v1-dependencies-{{ checksum "package.json" }}
+  build:
+    <<: *defaults
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            - v1-dependencies-
+
+      - run: npm install
+      - run: sudo npm install typescript -g
```

Comment:
I think we can install it without `sudo` nor `-g` and then run using `npx tsc`

Reply:
indeed @NickSeagull, I updated it 4d29c5bb9578a7c28eb44f7a7b7cde890ae5e8ef thanks